CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ option(USE_OPENCV "Use OpenCV" ON)
 option(USE_OPENMP "Use OpenMP for parallel code" OFF)
 option(USE_PROF "Use profiling" OFF)
 option(USE_REDIS "Use Redis" OFF)
+option(USE_QNNPACK "Use QNNPACK (quantized 8-bit operators)" ON)
```

Comment:
Nit: Move it one up so it's alphabetical?

Reply:
fixed